brainlit/utils/session.py

```diff
@@ -111,6 +111,7 @@ def set_url_segments(self, seg_url: str):
         self.url_segments = seg_url
         self.cv_segments = CloudVolume(self.url_segments, parallel=False)
 
+    # TOMMY REVIEW
     def get_segments(self, seg_id: int, bbox: Optional[Bounds] = None) -> nx.Graph:
```

Comment:
whats the type of output graph? networkx?

Reply:
Yes, if you look at the type hints it is clear. I will add a brief descriptor.